easybuild/easyconfigs/i/IPM/darshan-mk-log-dirs.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/perl -w
```

Comment:
ipm_parse is like pat_report and needs ploticus:

```
grep -m1 PLOTICUS_BIN software/IPM/2.0.6-CrayGNU-2016.11/bin/ipm_parse.pl
```
> chomp($PLOTICUS_BIN=`which pl`);

see dependencies  in IPM.eb